benefits/core/templates/core/base.html

```diff
@@ -71,14 +71,14 @@
       {% endblock main_content %}
     </main>
 
-    <footer id="footer" class="global-footer">
```

Comment:
Losing this class doesn't break anything from the State Template, right?

Reply:
As far as I've tested no.